node-fs-readfile/cat.js

```diff
@@ -0,0 +1,11 @@
+import { readFile } from 'node:fs/promises';
+
+const filenames = process.argv.slice(2);
+
+try {
+  const promises = filenames.map((filename) => readFile(filename, 'utf8'));
+  const contents = await Promise.all(promises);
+  console.log(contents);
```

Comment:
This logs the file contents as an array. To log as a single string, you can use `contents.join('\n');`.